src/Microsoft.ML.StandardLearners/Standard/Online/OnlineLearnerCatalog.cs

```diff
@@ -5,6 +5,7 @@
 using Microsoft.ML.Runtime;
 using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.Runtime.Learners;
+using Microsoft.ML.Trainers;
 using System;
 
 namespace Microsoft.ML
```

Comment:
>Microsoft.ML [](start = 10, length = 12)

should be Trainers right?

Reply:
or staticPipe

---
In reply to: [227479981](https://github.com/dotnet/machinelearning/pull/1340#discussion_r227479981) [](ancestors = 227479981)